pymarketstore/__init__.py

```diff
@@ -1,10 +1,11 @@
 from .client import Client  # noqa
-from .params import Params  # noqa
+from .params import Params, ListSymbolsFormat  # noqa
 from .jsonrpc_client import MsgpackRpcClient  # noqa
 from .grpc_client import GRPCClient  # noqa
 
 # alias
 Param = Params  # noqa
+ListSymbolsFormat = ListSymbolsFormat  # noqa
```

Comment:
nit: this is unnecessary if it's not renaming the enum (the import above is sufficient)